app/lib/account/default_auth_provider.dart

```diff
@@ -86,6 +93,73 @@ class DefaultAuthProvider extends BaseAuthProvider {
     final openIdData = await openIdDataFetch();
     return await token.verifySignature(openIdData.jwks);
   }
+
+  String _getOauthSiteAudience() =>
+      _oauthSiteAudience ?? activeConfiguration.pubSiteAudience!;
+
+  Future<String?> _getOauthSiteAudienceSecret(String audience) async {
+    return _oauthSiteAudienceSecret ??
+        await secretBackend.getCachedValue('${SecretKey.oauthPrefix}$audience');
```

Comment:
Does `secretBackend` cache this in-memory, or only on redis?

Reply:
Only in redis.